src/variables-tree.js

```diff
@@ -7,8 +8,19 @@ export function buildVariablesTree(variables, query) {
     if (!query) {
       return true
     }
-    const normalizedQuery = query.toLowerCase()
-    return variable.name.toLowerCase().includes(normalizedQuery)
+    function nameMatches(variable, query) {
+      const normalizedName = variable.name.toLowerCase()
+      return normalizedName.includes(query)
+    }
+    function labelMatches(variable, query) {
+      if (!variable.label) {
+        return false
+      }
+      const normalizedLabel = removeAccents(variable.label.toLowerCase())
```

Comment:
This implementation means that on every search, the whole tree is accent-folded… That's quite a lot of text!
I benched it, it's ok as long as the search is triggered only by an explicit action. Probably won't support search-as-you-type, but currently ok 😉